studio/components/interfaces/Functions/EdgeFunctionsListItem.tsx

```diff
@@ -21,7 +21,7 @@ const EdgeFunctionsListItem: FC<Props> = ({ function: item }) => {
   const { project } = useProjectContext()
   // get the .co or .net TLD from the restUrl
   const restUrl = project?.restUrl
-  const restUrlTld = new URL(restUrl as string).hostname.split('.').pop()
+  const restUrlTld = new URL(restUrl ?? 'http://example.com').hostname.split('.').pop()
```

Comment:
hmm i wouldn't use a placeholder here - lets do a check against whether restUrl is available 
```suggestion
  const restUrlTld = restUrl !== undefined ? new URL(restUrl as string).hostname.split('.').pop() : undefined
```